PLEP-0002.md

```diff
@@ -19,35 +19,37 @@ project and code development.
 
 ## Membership of Coordinating Committee
 
-### Limits on number of members
+### Membership
 
-The CC shall consist of at least 4 members and no more than 10
-members.  Within these limits, the CC may increase or decrease the
-number of directors serving on the CC, including for the purposes of
-staggering the terms of members.  The members of the Coordinating
-Committee shall be listed in [PLEP 
+The CC shall consist of at least 4 members and no more than 12 members.  
+Within these limits, the CC may increase or decrease the number of 
+directors serving on the CC, including for the purposes of staggering 
+the terms of members.  The CC should include not just core members but
+also members who work toward community engagement, education, and other
+goals of PlasmaPy.  Current and former members of the CC shall be listed 
```

Comment:
Actually, do we want PLEPs to be mutable once accepted and merged into this repository? If so, PLEP 3 may not be the best place to put that list. How do other projects do this?

Reply:
I do think that PLEPs should be mutable, as long as the changes are discussed and reasoning recorded.  Process PLEPs in particular should be living documents, since we will probably need to refine our procedure for releases fairly often, as an example.  Standard PLEPs may benefit from amendments if we learn something during their implementation that we didn't think of when they were accepted.  Having them be mutable gives us a lot more flexibility.  Informational PLEPs should be able to change when the information contained within them changes.

Python Enhancement Proposals are not mutable once they have been accepted or declined, I think.  SunPy allows amendments to SEPs, which is what I was largely using as a guide.  I'm glad that they put a CC BY 4.0 license on it. 👍

Reply:
Ah. That's quite reasonable then! I was basing this off of PEPs.